src/bio/bio_config.c

```diff
@@ -374,14 +377,6 @@ read_config(const char *config_file, struct json_config_ctx *ctx)
 	return rc;
 }
 
-static void
-free_json_config_ctx(struct json_config_ctx *ctx)
-{
-	D_FREE(ctx->values);
-	D_FREE(ctx->json_data);
-	D_FREE(ctx);
-}
-
```

Comment:
Why get rid of this in favor of more duplication?

Reply:
I felt it was more clear to move these to the 2 places they were used.